From 05469f7b42b6947d02ec9dd41fb08e777a2f22b9 Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Wed, 23 Jan 2013 01:02:14 +0100
Subject: [PATCH] man: clearify the meaning of timeout=0 for password agents

The fact that timeout=0 makes password agents wait indefinitely is documented
in http://www.freedesktop.org/wiki/Software/systemd/PasswordAgents. Document
it also in the relevant man pages.
(cherry picked from commit c93350728e822c3da7522664cf0a0e0ee10509fa)
---
 man/crypttab.xml             | 3 ++-
 man/systemd-ask-password.xml | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/man/crypttab.xml b/man/crypttab.xml
index 2a83994..acc7029 100644
--- a/man/crypttab.xml
+++ b/man/crypttab.xml
@@ -220,7 +220,8 @@
                                 for querying for a password. If no
                                 unit is specified seconds is used.
                                 Supported units are s, ms,
-                                us, min, h, d.</para></listitem>
+                                us, min, h, d. A timeout of 0 waits
+                                indefinitely.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
diff --git a/man/systemd-ask-password.xml b/man/systemd-ask-password.xml
index 7b0b9ab..563f9d5 100644
--- a/man/systemd-ask-password.xml
+++ b/man/systemd-ask-password.xml
@@ -129,7 +129,8 @@
 
                                 <listitem><para>Specify the query
                                 timeout in seconds. Defaults to
-                                90s.</para></listitem>
+                                90s. A timeout of 0 waits indefinitely.
+                                </para></listitem>
                         </varlistentry>
 
                         <varlistentry>
